src/lib/Preview.js

```diff
@@ -1524,7 +1524,7 @@ class Preview extends EventEmitter {
      * @private
      * @return {void}
      */
-    emitLoadMetrics() {
+    emitLoadMetrics({ encoding }) {
```

Comment:
It feels a little strange IMO to pass in an empty object to `emitLoadMetrics` to avoid an error destructuring `encoding` from the argument. Would it make sense to instead have logic to handle an undefined argument?

oh and jsdoc for the new method argument.